app/src/main/java/com/hammer/talkbbokki/data/remote/TalkbbokkiService.kt

```diff
@@ -1,5 +1,6 @@
 package com.hammer.talkbbokki.data.remote
 
+import com.hammer.talkbbokki.data.entity.*
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between